test/server/cards/01_SOR/units/BailOrganaRebelCouncilor.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('Bail Organa', function () {
+    integration(function () {
+        describe('Bail Organa\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['bail-organa#rebel-councilor', 'battlefield-marine'],
+                        spaceArena: ['red-three#unstoppable']
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors', 'cargo-juggernaut']
+                    }
+                });
+            });
+
+            it('should give an experience to an another ally', function () {
```

Comment:
nit, grammar: "should give an experience to an ally". "should give an Experience to an another friendly unit" is probably the clearest but either is fine